src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,612 @@
+
+MOLT LMS is self-hosted on [Kubernetes](https://kubernetes.io/) and [configured using Helm](#configuration). At a high level, the LMS consists of the following:
+
+- A number of proxy [instances](#lms-instances) (running in separate Kubernetes pods) across which application traffic is distributed and routed to the source and target databases.
+- An "orchestrator" service (running in a single Kubernetes pod) that coordinates the proxy instances and sends the cutover commands.
+
+This page describes how to [install](#installation), [configure](#configuration), [secure](#security), and [use the LMS](#molt-lms-cli) to perform a live migration. {% comment %}For more information, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).{% endcomment %}
+
+## Terminology
+
+- A *live migration* keeps two production databases online (a source and a target database) and uses either replication or dual writing to keep data equal between them until a final cutover.
+- The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
+- *Shadowing* is the replication of traffic on the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+
+## Requirements
+
+- [Kubernetes](https://kubernetes.io/) cluster
+- [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
+
+#### Supported databases
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
+- CockroachDB (source and target)
+
+## Installation
+
+To install the LMS, add the [Helm chart](https://molt.cockroachdb.com/lms/charts).
+
+When you install the `molt-lms` chart, the LMS proxy instances and orchestrator are initialized as Kubernetes pods:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+kubectl get pods
+~~~
+
+~~~
+NAME                                          READY   STATUS      RESTARTS      AGE
+lms-molt-lms-5cbd7c748-bzbgn                  1/1     Running     0             2m30s
+lms-molt-lms-5cbd7c748-g5zqf                  1/1     Running     0             2m34s
+lms-molt-lms-5cbd7c748-lbjjz                  1/1     Running     0             2m3
+lms-molt-lms-orchestrator-596d4b54d8-c4v97    1/1     Running     0             21m
+...
+~~~
+
+You will see `lms-molt-lms` pods that match the configured [number of LMS instances](#lms-instances), along with one `lms-molt-lms-orchestrator` pod.
```

Comment:
This is just a bit of a clarification thing, but the pod base name is just `molt-lms` and `molt-lms-orchestrator` the `lms` prefix at the front depends on what you named your release as that can vary. So might be worth calling that out.

Reply:
Noted. Which param name corresponds to this prefix?

Reply:
There isin't exactly field that sets the prefix.
If you set name override, that overrides the `molt-lms` or `molt-lms-orchestrator` part. So if I set name override to ryan. It would be `lms-ryan` and `lms-ryan-orchestrator`

If you set the full name override to say blah, it would be `blah` and blah-orchestrator`. I would just perhaps call out that in your example, our release name is `lms`.